data/mods/gen3/scripts.ts

```diff
@@ -475,6 +475,10 @@ export const Scripts: ModdedBattleScriptsData = {
 
 			return damage;
 		},
+		spreadMoveHit(targets, pokemon, move, hitEffect, isSecondary, isSelf) {
+			// Gen 4 overrides this method, so we need to revert it back
```

Comment:
Maybe the parent's `spreadMoveHit` function should just have an `if`, instead of this "override then override back" approach?

Reply:
In general, unless the code has very little in common (usually a gen 1-2 thing), I'd usually prefer `if (this.gen ...` rather than overriding back and forth.